### PR TITLE
[TR] Update Do you really want to ask peppy?

### PR DESCRIPTION
**Pull request by zeusminus**
Update Turkish translations for `Do_you_really_want_to_ask_peppy` article

### CODE REVIEW

File: wiki/Do_you_really_want_to_ask_peppy/tr.md [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
+# Gerçekten peppy'e sormak istiyor musunuz?
 
-# Do you really want to ask peppy?
+Selamlar! peppy sizin onunla derhal iletişime geçme isteğinizi anlıyor, ancak öncelikle sizden buranın ondan yardım ya da destek istemek için doğru bir yer **olmadığını** anlamanızı istiyor. Kendisi osu! desteğini kişisel mesajlardan ayrı tutmakta ve ayrıca forum özel mesajlarına ayak uydurmayı oldukça zor bulmaktadır. Bundan ötürü sizden aşağıdaki seçeneklerden birini yapmanızı istiyor:
 
-Selamlar. Benimle şahsi olarak iletişime geçme isteğinizi anlıyorum, ancak sizden buranın benden yardım/destek istemek için doğru yer **olmadığını** anlamanızı istiyorum. osu! desteği şahsi mesajlardan ayrı tutuyorum, ve ayrıca forum PM'lerine ayak uymayı oldukça zor buluyorum. Bundan ötürü sizden aşağıdakilerden birisini yapmanızı istiyorum:
+## Teknik Destek
 
-## Teknik destek
+Pek çok teknik sorun [Yardım forumunda](https://osu.ppy.sh/community/forums/5) arama yaparak veya gönderi oluşturarak çözülebilir. Orada size sorunlarınızda yardımcı olabilecek bir sürü insan var, dolayısıyla onunla iletişime geçmekten çok daha hızlı bir şekilde çözüm bulabilirsiniz.
```
Comment (goeo-) [inline wiki/Do_you_really_want_to_ask_peppy/tr.md:7]:
```suggestion
Çoğu teknik sorun [Yardım forumunda](https://osu.ppy.sh/community/forums/5) arama yaparak veya gönderi oluşturarak çözülebilir. Orada size sorunlarınızda yardımcı olabilecek bir sürü insan var, dolayısıyla onunla iletişime geçmekten çok daha hızlı bir şekilde çözüm bulabilirsiniz.
```

File: wiki/Do_you_really_want_to_ask_peppy/tr.md [hunk 2]
```diff
@@ -1,19 +1,15 @@
...
 
-Teknik sorunların büyük çoğunluğu [Tech Support forumunda](https://osu.ppy.sh/community/forums/5) arama yaparak ya da yeni başlık açarak çözüme kavuşturulabilir. Orada sorunlarınıza yardımcı olacabilecek bir sürü insan var, bu yüzden orada sorunlarınıza çözümünüzü benimle iletişime geçmekten çok daha hızlı bulabilirsiniz.
+## Genel osu! desteği
 
-## Genel osu! destek
-
-Hesap sorunları, ödemeler ile ilgili sorunlar, ban sonrası, turnuvalar hakkında sorgular ve **osu!'da yardım gerektiren herhangi bir şey** için, lütfen [accounts@ppy.sh](mailto:accounts@ppy.sh) adresine (hesap sorgulamaları) ya da [osu@ppy.sh](mailto:osu@ppy.sh) (geri kalan her şey) bir e-posta gönderin.
+Hesap sorunları, ödeme problemleri, ban takipleri, turnuvalar hakkındaki sorgular ve **osu!'da yardım gerektiren herhangi bir şey** için, lütfen [accounts@ppy.sh](mailto:accounts@ppy.sh) adresine (hesap sorguları) ya da [osu@ppy.sh](mailto:osu@ppy.sh) adresine (geri kalan her şey) e-posta gönderin.
```
Comment (goeo-) [inline wiki/Do_you_really_want_to_ask_peppy/tr.md:11]: `sorgu` feels bad here. not sure about a replacement

File: wiki/Do_you_really_want_to_ask_peppy/tr.md [hunk 3]
```diff
@@ -1,19 +1,15 @@
...
 
 ## peppy'e ulaşmak
 
-Eğer peppy'e osu! dışındaki sebeplerden ötürü ulaşmak istiyorsanız, ona [pe@ppy.sh](mailto:pe@ppy.sh) adresinden ulaşabilirsiniz. Bu e-posta adresine gönderilen hesap ya da destek konulu bütün e-postaların spam olarak değerlendirileceğini biliniz.
+Eğer osu! ile alakalı olmayan sebeplerden dolayı peppy ile kişisel olarak iletişime geçmek istiyorsanız, [pe@ppy.sh](mailto:pe@ppy.sh) adresinden ulaşabilirsiniz. Bu e-posta adresine gönderilen hesap ile ilgili ya da destek konulu bütün taleplerin spam olarak işaretleneceğini biliniz.
```
Comment (goeo-) [inline wiki/Do_you_really_want_to_ask_peppy/tr.md:15]:
```suggestion
Eğer osu! ile alakalı olmayan sebeplerden dolayı peppy ile kişisel olarak iletişime geçmek istiyorsanız, ona [pe@ppy.sh](mailto:pe@ppy.sh) adresinden ulaşabilirsiniz. Bu e-posta adresine gönderilen destek taleplerinin ve osu! hesaplarıyla ilgili e-postaların spam olarak işaretleneceğini lütfen aklınızda bulundurunuz.
```

File: wiki/Do_you_really_want_to_ask_peppy/tr.md (hunk 2 above)
Comment (zeusminus) [reply wiki/Do_you_really_want_to_ask_peppy/tr.md:11]:
```suggestion
Hesap sorunları, ödeme problemleri, ban takipleri, turnuvalar ile ilgili sorular ve **osu!'da yardım gerektiren herhangi bir şey** için, lütfen [accounts@ppy.sh](mailto:accounts@ppy.sh) adresine (hesap konulu sorular) ya da [osu@ppy.sh](mailto:osu@ppy.sh) adresine (geri kalan her şey) e-posta gönderin.
```
perhaps it sounds better like this?

though I still feel like there's nothing that feels odd when `sorgu` is used for the latter, this change definitely suits well with the former

**Pull request merged**